src/App.tsx

```diff
@@ -1,51 +1,15 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+import { Outlet } from 'react-router-dom';
+import NavBar from './Components/NavBar';
```

Comment:
This import will not work, `Components` folder starts with lowercase letter.
Is this the same code that is published to Demo link?
![image](https://github.com/mate-academy/react_tabs-with-router/assets/23013525/845624ef-1e78-46cd-849d-e51e4570c230)